extensions/aria_extension_tosca/simple_v1_0/modeling/__init__.py

```diff
@@ -446,6 +451,29 @@ def get(name):
     return model
 
 
+def create_workflow_operation_template_model(context, service_template, policy):
+    model = OperationTemplate(name=policy._name,
```

Comment:
Do we want workflows to be modeled as OperationTemplates?
There are indeed many similarities though some fields don't really apply, e.g. `retry_interval`; What do you think?